lib/node_modules/@stdlib/math/base/special/cabs/README.md

```diff
@@ -154,19 +156,28 @@ double stdlib_base_cabs( const double complex z );
 
 ```c
 #include "stdlib/math/base/special/cabs.h"
+#include "stdlib/complex/float64.h"
+#include "stdlib/complex/reim.h"
 #include <stdio.h>
-#include <complex.h>
 
 int main() {
-    double complex x[] = { 3.14+1.0*I, -3.14-1.0*I, 0.0+0.0*I, 0.0/0.0+0.0/0.0*I };
-
-    double complex v;
+    stdlib_complex128_t x[] = {
```

Comment:
```suggestion
    const stdlib_complex128_t x[] = {
```

We recently upgraded `cppcheck` and it now raises lint errors here, as `x` can be an array of constants. https://github.com/stdlib-js/stdlib/actions/runs/4168623999/jobs/7215647598